lgc/include/lgc/state/PipelineState.h

```diff
@@ -300,10 +300,16 @@ class PipelineState final : public Pipeline {
   void initializeInOutPackState();
 
   // Get whehter the input locations of the specified shader stage can be packed
-  bool canPackInput(ShaderStage shaderStage) const { return m_inputPackState[shaderStage]; }
+  bool canPackInput(ShaderStage shaderStage);
```

Comment:
Can we still make these two methods as constant functions?

Reply:
If keep the constant property, `canPackInput` cannot call non-const function `getShaderResourceUsage` to check if it is a null FS.